docs/src/user-docs/guides-multi-node.md

```diff
@@ -3,8 +3,310 @@
 A multi-node deployment allows you to run CLP across a distributed set of hosts.
 
 :::{warning}
-CLP now uses [Docker Compose][docker-compose] for orchestration and support for multi-node
-deployments has been removed temporarily.
+The instructions below provide a temporary solution for multi-node deployment using manual Docker
+Compose orchestration and may change as we actively work to improve ease of deployment. Stay tuned
+for future updates on Kubernetes Helm support, which will simplify multi-node deployments
+significantly.
 :::
 
+## Requirements
+
+* [Docker] and [Docker Compose][docker-compose]
+  * If you're not running as root, ensure Docker can be run
+    [without superuser privileges][docker-non-root].
+* One or more hosts networked together
+* When not using S3 storage, a shared filesystem accessible by all worker hosts (e.g., NFS,
```

Comment:
just realize that seaweedfs is accessible via s3 too. maybe it's better to say "cloud s3" or "aws s3" to avoid confusions. or we can just say it's possible to set up an s3 service with seaweedfs too